entry_types/scrolled/doc/creating_themes.md

```diff
@@ -19,3 +19,7 @@ Register the theme in the Pageflow initializer:
     end
 
 Copy all options from the default theme and make the desired changes.
+
+Place the favicons next to logos in the theme. Please create favicons
+according to the devices for performance reasons and replace them with
+the default favicons.
```

Comment:
> Place the favicons next to logos in the theme.

**suggestion**: Place favicon files in sub directory of theme directory

Among all these favicon files, other theme images are hard to spot. Since all these files are about the same thing and are probably generated at once, I suggest placing them in a `favicons` sub directory of the theme directory.

> Please create favicons ￼according to the devices for performance reasons

**question**: What is the recommended way to generate these files?

Did you use one of the favicon generators? I think it would be handy if the file names used here matched up with what one of those services generates. We could then name that service here.

Reply:
Placed the favicons in a sub-directory. 

Yes, the source that you mentioned in the ticket is the one. It is really helpful and creates all these files, you can further edit those files like background, and border.
Furthermore, the names are just as they are mentioned when generated and the source also gives all the html tags to include as well using the same names.

I will name the source and give a link in the docs.